.env.example

```diff
@@ -72,9 +72,16 @@ ENABLE_WW1_FULLTEXT_SEARCH=0
 # If absent, Disqus comments will be disabled.
 DISQUS_SHORTNAME=shortname-here
 
-# Enable authentication for User galleries (XX), enabled by default
-ENABLE_XX_USER_AUTH=1
-OAUTH_URL=https://keycloak-test.eanadev.org/auth/realms/europeana/protocol/openid-connect
+# Enable authentication for User galleries (XX). Disabled by default
+# ENABLE_XX_USER_AUTH=0
+# Keycloak server URL
+# OAUTH_URL=https://keycloak-test.eanadev.org/auth/realms/
+# Auth module config options for Keycloak
+# OAUTH_SCHEME=oauth2
+# OAUTH_REALM=europeana
+# OAUTH_CLIENT=collections_portal
+# OAUTH_SCOPE=[openid, profile, email, usersets]
```

Comment:
This results in a string with value `"[openid, profile, email, usersets]"`.

Change the example setting here to:
```
# OAUTH_SCOPE=openid,profile,email,usersets
```

And when utilising it elsewhere, parse with:
```
process.env.OAUTH_SCOPE.split(',')
```